package.json

```diff
@@ -28,6 +27,7 @@
   ],
   "devDependencies": {
     "chai": "3.5.0",
+    "flow-bin": "0.33.0",
```

Comment:
greenkeeper seems to make pull requests on all releases even with a caret but pinning this to an exact version will be good for development so that others will know what worked on another person's machine :)